src/DependencyInjection/M6WebGuzzleHttpExtension.php

```diff
@@ -213,6 +213,11 @@ protected function getCurlConfig(array $config)
 
     protected function getServiceReference(ContainerBuilder $container, $id)
     {
+        // if $id is already a Reference (e.g. if parts of the config are defined as parameters), return it
+        if ($id instanceof Reference) {
```

Comment:
So, is this function still relevant? Because if it starts with an `@`, it will be already an actual reference, and if not we return `null`…